### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jcohen**

### CODE REVIEW

**Comment by pdokas** [comment]
:+1:

**Comment by saw** [comment]
:shipit:

**Comment by pdokas** [comment]
+:100:

**Comment by superic** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -7,4 +7,6 @@ Interested in cloning the site for your own interviews, similar or otherwise? HE
 	3) And what software?
 	4) What would be your dream setup?
 
-If you *do* make your own version, please send me a note - I'd love to add it to the Community page!
\ No newline at end of file
+If you *do* make your own version, please send me a note - I'd love to add it to the Community page!
+
+butts.
```
Comment (henrylyne) [inline README.md]: Please use an upper-case B.

Comment (waferbaby) [inline README.md]: Could you refactor this to use the ButtsFactory class, so we don't have to keep typing the word out?

Comment (saw) [inline README.md]: Would this support other body parts? How about creating a body part interface, then people can implement various body parts as required.

**Comment by pdokas** [comment]
You monster.

**Pull request closed without merging**